src/applications/vaos/tests/mocks/helpers.js

```diff
@@ -195,17 +195,42 @@ export function mockPastAppointmentInfoOption1({ va = [], cc = [] }) {
  *
  * @export
  * @param {Object} params
- * @param {Array<string>} ids List of facility ids to use in query param
- * @param {Array<VAFacility>} facilities Array of facilities to return from mock
+ * @param {?Array<string>} ids List of facility ids to use in query param
+ * @param {?Array<VAFacility>} facilities Array of facilities to return from mock
  */
 export function mockFacilitiesFetch(ids, facilities) {
-  setFetchJSONResponse(
+  if (!ids) {
+    setFetchJSONResponse(
+      global.fetch.withArgs(sinon.match('/v1/facilities/va?ids=')),
```

Comment:
As mentioned above, if no args are passed, just mock any facilities call with this substring. This gets overridden by later mocks, so it doesn't break anything in tests that have specific facility data they mock.